app/Controllers/Account.php

```diff
@@ -12,8 +13,15 @@
 
 class Account extends BaseController
 {
+    const REGISTER_RULES = [
+        'username' => 'required|trim|alpha_dash|min_length[3]|max_length[30]',
+        'password' => 'required|valid_password',
+        'email' => 'required|trim|valid_email|max_length[320]',
+        'token' => 'permit_empty|trim|alpha_numeric|max_length[128]',
+    ];
+
     const VERIFICATION_RULES = [
```

Comment:
Where do you need this?
I can only see, that you use the `REGISTER_RULES`.

Reply:
I use them here: https://github.com/TechStreamConference/website-backend/blob/101-allow-registering-for-existing-user-using-a-predefined-token/app/Controllers/Account.php#L184

Reply:
Ah I see